modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -58,6 +66,52 @@ public String getFilesystemFriendlyName() {
                 assertEquals("Recorded file not found", 1, files.length);
             }
         }
+
+        @Test
+        public void recordingTestThatShouldHaveCorrectDuration() throws IOException, InterruptedException {
+            final String flvFileTitle = "ChromeThatRecordsAllTests-recordingTestThatShouldBeRecordedAndRetained";
+            final String flvFileNameRegEx = "PASSED-" + flvFileTitle + ".*\\.flv";
+            DockerImageName chromeDockerImageName = DockerImageName.parse("selenium/standalone-chrome-debug")
+                                                             .withTag(SeleniumUtils.determineClasspathSeleniumVersion());
+            try (
+                BrowserWebDriverContainer chrome = new BrowserWebDriverContainer(chromeDockerImageName)
+                    .withRecordingMode(RECORD_ALL, vncRecordingDirectory.getRoot())
+                    .withRecordingFileFactory(new DefaultRecordingFileFactory())
+            ) {
+                chrome.start();
+
+                doSimpleExplore(chrome);
+                chrome.afterTest(new TestDescription() {
+                    @Override
+                    public String getTestId() {
+                        return getFilesystemFriendlyName();
+                    }
+
+                    @Override
+                    public String getFilesystemFriendlyName() {
+                        return flvFileTitle;
+                    }
+                }, Optional.empty());
+
+                String recordedFile = vncRecordingDirectory.getRoot().listFiles(new PatternFilenameFilter(flvFileNameRegEx))[0].getCanonicalPath();
+
+                String dockerOutput;
```

Comment:
```suggestion
                String dockerOutput = "";
```

Maybe better to initialize with empty String, so we can't end up with NPE in the assert.

Reply:
It's no longer the case, I merged the declaration and the assignment, thanks :thumbsup: